src/backend/src/bitcoin_utils.rs

```diff
@@ -0,0 +1,32 @@
+use std::str::FromStr;
+use ic_cdk::api::management_canister::bitcoin::{BitcoinNetwork};
+
+fn parse_derivation_path(path: &str) -> Vec<Vec<u8>> {
+    path.split('/')
```

Comment:
should we assert the expected format of `path`? To some extension it can be misused by passing anything as input string some day.

Reply:
agreed